### PR TITLE
doc(service): add name to nodePort

### PR DESCRIPTION
**Pull request by udservaites**
nodePort requires a name, if it is missing, you will get an error

### CODE REVIEW

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -508,7 +508,8 @@ spec:
   selector:
     app.kubernetes.io/name: MyApp
   ports:
-    - port: 80
+    - name: name-of-node-port
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md:511]:
```suggestion
    - name: http
```

Thanks, but: there is an official name for (TCP) port 80; we should use that.

**Review by sftim — COMMENTED** [review]
/sig network

Let's get a technical review on this advice.

**Comment by adityasamant25** [comment]
> nodePort requires a name, if it is missing, you will get an error

@udservaites I tried to created a NodePort service using the manifest described [here](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport-custom-port).

I did not receive any error and the service was created.

If you have some example or evidence to demonstrate that `name` is a mandatory field under `ports`, please share it.

Page in context: https://kubernetes.io/docs/concepts/services-networking/service/

**Comment by sftim** [comment]
This seems to be more about Services with multiple ports than about Services of type NodePort.

https://kubernetes.io/docs/concepts/services-networking/service/#multi-port-services already says:
> When using multiple ports for a Service, you must give all of your ports names so that these are unambiguous. 

(and this is true)

/hold
The document we already have is accurate and I don't see grounds to change it.

**Comment by sftim** [comment]
/hold cancel

/close
We don't need to make a change here

**Pull request closed without merging**